java/README.rst

```diff
@@ -92,304 +91,9 @@ correspondent calls executed on remote machines.
         }
     }
 
-Ray Java API
+More information
 ============
 
-Basic API
----------
+- `API document <https://github.com/ray-project/ray/tree/master/java/api_doc.rst>`_
+- `Tutorial <https://github.com/ray-project/ray/tree/master/java/tutorial>`_
```

Comment:
How about direct to the README.rst in tutorial package? And before the commit is check in the url maybe not worked.

Reply:
I think directing to tutorial package (instead of tutorial/README.rst) would be  better. Because it also shows the code files, so readers can easily navigate to the code by a few clicks. 
See, https://github.com/ant-tech-alliance/ray/tree/java_tutorial/java/tutorial